week3/menu-recommend/src/components/MainSection/SelectArticle/index.jsx

```diff
@@ -0,0 +1,44 @@
+import * as S from './style';
+
+const SelectArticle = ({ selectOption, selectOptionHandler, startPickHandler }) => {
+  return (
+    <>
+      <S.SubTitleStyle>원하는 추천 방식을 골라줘!</S.SubTitleStyle>
+
+      <S.MainSectionStyle>
+        {/* 초기 렌더링혹은 취향대로 추천 선택시 표시 */}
+        {(selectOption === '' || selectOption === 'personal') && (
+          <S.ArticleStyle
+            onClick={() => {
+              selectOptionHandler('personal');
+            }}
+          >
+            <a href="#">취향대로 추천</a>
+          </S.ArticleStyle>
+        )}
+        {/* 초기 렌더링 혹은 랜덤 추천 선택시 표시 */}
+        {(selectOption === '' || selectOption === 'random') && (
+          <S.ArticleStyle
+            onClick={() => {
+              selectOptionHandler('random');
+            }}
+          >
+            <a href="#">랜덤 추천</a>
+          </S.ArticleStyle>
+        )}
+      </S.MainSectionStyle>
```

Comment:
여기 실제 태그로 보면 `article` 에 onClick을 주고 안에서 `a` 태그로 상단 이동이 되는 구조 같은데, 이렇게 구현한 이유가 있을까?! 

나는 랜덤추천과 취향대로 추천이 각각 버튼이다 보니
```suggestion
        {(selectOption === '' || selectOption === 'random') && (
          <button
            onClick={() => {
              selectOptionHandler('random');
            }}
          >랜덤 추천
          </button>
        )}
```
이렇게 button태그를 쓰는게 더 자연스럽다고 생각했는데 !